docs/devices/E1766.md

```diff
@@ -48,6 +48,7 @@ When connected, the light turns off.
 ### Binding
 The [binding](../guide/usage/binding.md) functionallity of this remote varies per firmware version:
 - below 2.3.75: suppports binding to groups only. It can only be bound to 1 group at a time. By default this remote is bound to the default bind group which you first have to unbind it from. This can be done by sending to `zigbee2mqtt/bridge/request/device/unbind` payload `{"from": "DEVICE_FRIENDLY_NAME", "to": "default_bind_group"}`. Wake up the device right before sending the commands by pressing a button on it.
+- Devices on 2.3.079 appear to support binding to devices.
 - 2.3.75 and greater: supports binding to devices only
```

Comment:
`- Devices on 2.3.079 appear to support binding to devices.` is the same as `- 2.3.75 and greater: supports binding to devices only`? Or am I overlooking something.

Reply:
Is 2.3.75 not > 2.3.079 or should I be reading it as 2.3.79?

Reply:
`2.3.75` should be `2.3.075` I think

Reply:
That makes more sense I think

Reply:
Fixed, can this be closed now?